platform/ios/README.md

```diff
@@ -4,9 +4,10 @@
 
 A library based on [Mapbox GL Native](../../README.md) for embedding interactive map views with scalable, customizable vector maps into Cocoa Touch applications on iOS 8.0 and above using Objective-C, Swift, or Interface Builder.
 
-This repository is for day-to-day development of the SDK. Building the SDK yourself requires [a number of dependencies and steps](../../INSTALL.md) that are unnecessary for developing production applications. For production applications, please consider installing an official, prebuilt release instead; see the [Mapbox iOS SDK website](https://www.mapbox.com/ios-sdk/) for installation instructions.
+This repository is for day-to-day development of the SDK. Building the SDK yourself requires [a number of dependencies and steps](../../INSTALL.md) that are unnecessary for developing production applications. For production applications, please consider installing an official, production-ready version instead.
 
-* [Integrating the Mapbox iOS SDK into your application](INSTALL.md)
+* [Integrate a production-ready version of the Mapbox iOS SDK into your application](https://www.mapbox.com/install/ios/).
+* [Integrate custom builds of the Mapbox iOS SDK into your application](INSTALL.md)
```

Comment:
I think ideally we would always encourage the pre-built versions to put "into your application". Maybe we could word this in a way that would frame it more as "developing and contributing" to the project? 

"Build, develop, and contribute to the Mapbox iOS SDK" ?

I know that conflicts with the description for DEVELOPING.md just below, but if you read it, `DEVELOPING.md` is almost completely about tips and tricks for modern C++ development. I think INSTALL.md gives iOS developers more useful information about how to get set up to contribute to the project.